code/game/machinery/kitchen/microwave.dm

```diff
@@ -51,7 +53,7 @@
 		// This will do until I can think of a fun recipe to use dionaea in -
 		// will also allow anything using the holder item to be microwaved into
 		// impure carbon. ~Z
-		acceptable_items |= /obj/item/weapon/holder
+		acceptable_items |= /obj/item/weapon/holder // I Can't Believe It's Hacky(tm)

```

Comment:
Go ahead and tell me how this is any hackier than any other method of making holders microwaveable.

Reply:
more of a complaint about microwave code in general but i have a Feeling(TM) there's a better way to do this